src/content/patterns/chatbot/content.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Chatbots
+tabs: ['Overview', 'Usage', 'Content']
+---
+
+### You have established a need for a chatbot based on extensive market validation and user research. You have also determined the scope of your bot and the topics it will be able to discuss with users. How should you begin to think about designing your conversation?
```

Comment:
```suggestion
### You have established a need for a chatbot based on extensive market validation and user research. How should you begin to think about designing your conversation?
```

This was a bit long as a header. I think we can drop the middle sentence and still have the same impact.